frontend/package.json

```diff
@@ -7,11 +7,13 @@
     "deploy": "yarn build && yarn export",
     "dev": "next dev",
     "export": "next export",
+    "fix-lint": "eslint . --fix",
```

Comment:
My preference is to actually include this directly into lint, so that any time a contributor runs "lint" they automatically fix. I know that some people don't like that though so added this. Wdyt @positiveimpact?

Reply:
I have never tried this feature so I'm not sure how reliable it is. After all, we can still reverse a change it makes before committing. Still, I'd leave it as the `fix-lint` command for now and possibly move it to `lint` later.